homeassistant/components/bluetooth/__init__.py

```diff
@@ -75,20 +107,32 @@ class BluetoothScanningMode(Enum):
 
 
 BluetoothChange = Enum("BluetoothChange", "ADVERTISEMENT")
-BluetoothCallback = Callable[[BluetoothServiceInfo, BluetoothChange], None]
+BluetoothCallback = Callable[[BluetoothServiceInfoBleak, BluetoothChange], None]
```

Comment:
Should we really expose this subclass in the callnack? Or should it not just be a potential bleak subclass (ie either a bleak variant or standard type)

Reply:
I'll make it a union so its clear it might not be there at some point